### ISSUE TITLE
Terminal font rendering problem

### ISSUE THREAD

**Issue by Photonico** [microsoft/vscode#148485]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.66.2
- OS Version: openSUSE Tumbleweed
- Theme: [Photonica](https://marketplace.visualstudio.com/items?itemName=ConAntares.Photonica)
- Font: FiraMono Nerd Font

Steps to Reproduce:

The font rendering is not correct in VSCode's terminal.
However, it works well in Konsole.

in VSCode:
![](https://raw.githubusercontent.com/Photonico/Problems/master/2022/Screenshot_20220501_033345.png)
![](https://raw.githubusercontent.com/Photonico/Problems/master/2022/Screenshot_20220501_034630.png)

in Konsole
![](https://raw.githubusercontent.com/Photonico/Problems/master/2022/Screenshot_20220501_033504.png)

**Comment by albertosantini** [comment]
TLDR;  "terminal.integrated.minimumContrastRatio": 1,

See https://code.visualstudio.com/updates/v1_66#_minimum-contrast-ratio-default-changed

**Comment by yogesh-aggarwal** [comment]
> TLDR; `"terminal.integrated.minimumContrastRatio": 1`
> 
> See https://code.visualstudio.com/updates/v1_66#_minimum-contrast-ratio-default-changed

edq
